src/app/record/primitive-record/primitive-record.component.scss

```diff
@@ -0,0 +1,30 @@
+.value-text{
+    color: blue;
+}
+
+.collapsable-control{
```

Comment:
duplicate also, you should put all this shared css in a file add to `styleUrls` of each component.

Reply:
space after class name